src/podman.scss

```diff
@@ -24,6 +24,12 @@
     .pf-m-grid-md.pf-c-table [data-label] {
         justify-items: start;
     }
+
+    // Limit details width to 80 characters or 2/3 the width of the page,
+    // whichever is larger
+    .pf-c-description-list:not(:only-child) {
```

Comment:
I can't see why the default wrapping does not work - please let's clarify this with a screenshot before merging.

Reply:
This is to allow the command to wrap when needed, so the layout flows as intended.

This is what it looks like on main:
![Screenshot 2021-09-14 at 17-35-53 Podman containers - garrett Bolt](https://user-images.githubusercontent.com/10246/133289707-edb9bfac-55ff-4262-91fd-27b7e124ed2d.png)

And here's this PR:
![Screenshot 2021-09-14 at 17-31-43 Podman containers - garrett Bolt](https://user-images.githubusercontent.com/10246/133289790-75b3f972-723e-42aa-a83a-7134f7c9a4b1.png)

You can see the command line is really long in this case, so this tells it to wrap when it needs to, so the layout works. The only child limiting is saying that if there are no other details, then this one can take the whole space.

Reply:
Additionally, in smaller sizes (this is a narrower desktop mode with just the content, but it applies to mobile too), the layout is messed up as it doesn't use gap but tries to fake things with margins:

On main:
![Screenshot 2021-09-14 at 17-36-12 Podman containers - garrett Bolt](https://user-images.githubusercontent.com/10246/133290264-8261acb5-d0c9-4021-af2e-50f42555868b.png)

With this PR:
![Screenshot 2021-09-14 at 17-38-22 Podman containers - garrett Bolt](https://user-images.githubusercontent.com/10246/133290326-3de57257-96ea-4903-b207-1d4e3cd60757.png)

Reply:
I reverted this commit - and kept all others in this PR  and I can get the issues you see here with long commands.